library/Icingadb/Widget/Detail/ObjectDetail.php

```diff
@@ -300,22 +300,12 @@ protected function createNotifications()
 
     protected function createPerformanceData()
     {
-        require_once Icinga::app()->getModuleManager()->getModule('monitoring')->getBaseDir()
-            . '/application/views/helpers/Perfdata.php';
-
-        $helper = new Zend_View_Helper_Perfdata();
-        $helper->view = Icinga::app()->getViewRenderer()->view;
-
         $content[] = Html::tag('h2', t('Performance Data'));
 
         if (empty($this->object->state->performance_data)) {
             $content[] = new EmptyState(t('No performance data available.'));
         } else {
-            $content[] = new HtmlElement(
-                'div',
-                ['id' => 'check-perfdata-' . $this->object->checkcommand],
```

Comment:
Please re-add the id. It's necessary for the collapsible state to persist correctly.